README.md

```diff
@@ -149,16 +148,15 @@ configured by setting the `manager` option to `subscription_manager`:
 
 ```ruby
 Vagrant.configure('2') do |config|
-...
+  ...
   if Vagrant.has_plugin?('vagrant-registration')
     config.registration.manager = 'subscription_manager'
   end
-...
+  ...
 end
 ```
 
-In case of `subscription_manager` manager for the preferred registration pair,
-you would be ask on your username/password combination.
+In case of you have choosen `subscription_manager` as manager, you would be asked for your username/password combination.
 
 vagrant-registration supports all the options of subscription-manager's register command.
 You can set any option easily by setting `config.registration.OPTION_NAME = 'OPTION_VALUE'`
```

Comment:
The vagrant-registration plugin supports all the options for the subscription-manager's register command.